src/main/java/ca/utoronto/fitbook/application/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package ca.utoronto.fitbook.application.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+// The error thrown given a username not in the database
+@ResponseStatus(value= HttpStatus.UNPROCESSABLE_ENTITY, reason="User Not Found")
+public  class UserNotFoundException extends RuntimeException {
```

Comment:
Exception already exists in master branch. Rebase onto master branch if missing if already rebased reuse exception.